### ISSUE TITLE
Need some help with loading the graph

### ISSUE THREAD

**Issue by stardust66** [ChristopherChudzicki/math3d#65]
So @ChristopherChudzicki if you look on http://math3d.herokuapp.com you'll find the latest version of the app. Logging in, registering and saving graphs all work, and you can play around with that. The names of graphs appear in the Graphs tab I added. The problem is I don't know how to load a graph after the math3d object is already initialized. I tried
```
math3d = new Math3D(Math3D.decodeSettingsAsURL64(settings));
```
but that didn't work. The loading screen appears but the original scene is still there. Is there a way to remove the original scene first?

Also, a big change that happened was that I had to change all the angular tags from {{}} to [[]], because they conflict with Flask tags. I think I changed them in all the places I know of and nothing seems to be breaking. Please tell me if you notice anything.

**Comment by ChristopherChudzicki** [comment]
I was wondering what would happen to angular `{{}}` now that we are using flask. Cool, I didn't realize `[[]]` works for angular also.

Nice job on Heroku app!! I made an account (and will make more) but haven't tested it much yet.

In the past, loading new graphs has always happened when the page loads so what you describe hasn't been an issue. I agree that loading new graphs from a saved graph menu without reloading entire page is much nicer.

The underlying graphics library that math3d uses is [mathbox](https://github.com/unconed/mathbox). Mathbox uses a DOM similar to HTML (but created in javascript) and has a selection API similar to JQuery. You can see the mathbox DOM print in the console on page load (or by `math3d.mathbox.print()`.

I think what we want is to destroy the original mathbox object and create a new one. I can't figure out how to destroy the original mathbox object, but I know how to entirely clear its DOM tree. Some refactoring of the mathbox class should make possible:

```
math3d = new math3d()
math3d.load(settings0)
math3d.clear()
math3d.load(settings1)
```

Would that do what you need?

**Comment by stardust66** [comment]
Yes, that would work. Thank you. By the way, the code to change the brackets is:
```
app.config(function($interpolateProvider) {
    $interpolateProvider.startSymbol('[[');
    $interpolateProvider.endSymbol(']]');
}
```
For future reference. Angular seems like a really cool library.

**Comment by stardust66** [comment]
Solved by #66.
